compiler/test/compilable/test23705.d

```diff
@@ -0,0 +1,8 @@
+// https://issues.dlang.org/show_bug.cgi?id=23705
+
+// DISABLED: win32
+
+void main ()
+{
+    ubyte [0x7fff_fffe] x;
```

Comment:
This test fails on many platforms - especially i686 and other 32-bit architectures - as there is no space for the frame pointer.

Reply:
I don't think so, but perhaps `static if (size_t.sizeof > 4)` could be added.

Reply:
FAOD, GCC enforces that the maximum size for local variables to occupy the stack to be:
```
ubyte [(1 << (size_t.sizeof * 8 - 1)) - 64 * size_t.sizeof] x;
```
That is per function, not per variable.